tests/e2e/bento_server_http/tests/test_serve.py

```diff
@@ -0,0 +1,15 @@
+import bentoml
+
+
+def test_http_server(bentoml_home: str):
```

Comment:
lets also add test for `server.start()` as a context manager as well

Reply:
Should be added now!